cob_hardware_config/robots/cob4/config/pc_monitor_cob4-b1.yaml

```diff
@@ -1,4 +1,11 @@
-check_core_temps: False
 check_ipmi_tool: False
-enforce_clock_speed: False
+check_core_temps: True
 num_cores: 4
+core_load_warn: 90
+core_load_error: 110
+load1_threshold: 10
+load5_threshold: 10
+core_temp_warn: 95 #Celsius
```

Comment:
we might need to raise the limits as for other robots

Reply:
it's yaml...can be done whenever needed....